src/utilities/IOManager.cpp

```diff
@@ -91,6 +92,13 @@ const proto::SSL_Referee &IOManager::getRefereeData() {
     return *msg;
 }
 
+const std::unordered_map<uint8_t, proto::RobotFeedback> &IOManager::getFeedbackDataMap() {
```

Comment:
This whole function is a hack, either return a copy of the internal world, aka remove const&, or return a const& to the internal map, dont heap allocate and then return a reference to the heap allocated object